src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -1019,6 +1039,33 @@ int sd_dhcp_server_start(sd_dhcp_server *server) {
                 return r;
         }
 
+
+        if (!server->bind_to_interface) {
+                r = dhcp_network_bind_udp_socket(server->ifindex, INADDR_BROADCAST, DHCP_PORT_SERVER, -1);
+
```

Comment:
please drop this empty line, we generally try to keep function calls and their immediate error handling together, which follows here, hence there should not be an empty line in-between